server/src/controllers/blog-review.js

```diff
@@ -0,0 +1,47 @@
+import Blog from '../model/blog.js';
+
+export const reviewBlog = async (req, res) => {
+  const { blogId } = req.params;
+  const { reviewStatus, reviewMessage } = req.body;
+
+
+  try {
+  const blog = await Blog.findById(blogId);
+  if (!blog) {
+    return res.status(404).json({ message: 'Blog not found' });
+  }
+
+  const user = req.user; 
```

Comment:
implemented in a later stage, not required right now. Will cause the api to fail